src/types/algorand-client.ts

```diff
@@ -118,8 +135,13 @@ export class AlgorandClient {
     return this._accountManager
   }
 
+  /** Methods for interacting with assets. */
+  public get asset() {
+    return this._assetManager
```

Comment:
Added new manager (assetManager). For now it's simple (single method), but over time it may grow in complexity.

Reply:
Should the bulk functions go in the AssetManager?